.github/workflows/continuous_integration.yaml

```
@@ -0,0 +1,108 @@
+      - main
+      - master
+  workflow_dispatch:
+
+permissions:
+  contents: write 
+
+jobs: 
+    continuous_integration:
+      runs-on: ubuntu-latest
+      env:
+          DOCKER_USERNAME: joakim077
```

YAMLlint (1.35.1)

[warning] wrong indentation: expected 8 but found 10

(indentation)